src/app/pages/pages-and-posts/TransformStore.js

```diff
@@ -0,0 +1,35 @@
+import { Button, Card, Title } from '@newfold/ui-component-library';
+import { ReactComponent as TransformLogo } from '../../../../assets/svg/transformStore.svg';
```

Comment:
Please use the `Assets` path alias to simplify the import path.

Reply:
tried it but tests are failing as it is not able to find the path.